platforms/documentation/docs/src/snippets/multiproject/buildSrc/groovy/services/build.gradle

```diff
@@ -0,0 +1,38 @@
+// tag::non-common-logic[]
+plugins {
+    id("java-common-conventions")
+}
+
+dependencies {
+    implementation("com.google.guava:guava:32.1.2-jre")
+}
+// end::non-common-logic[]
+
+/*
```

Comment:
Is it intentional to have lines 11-38 commented out?

Reply:
Yes, but I agree it's not ideal. Fixed as a separate snippet.